framework/SupervisedLearning/ARMA.py

```diff
@@ -264,10 +264,7 @@ def __trainLocal__(self,featureVals,targetVals):
       self._signalStorage[target]['original'] = copy.deepcopy(timeSeriesData)
       # if we're enforcing the training CDF, we should store it now
       if self.preserveInputCDF:
-        counts, edges = np.histogram(timeSeriesData, bins=self._computeNumberOfBins(timeSeriesData), density=False)
-        counts = np.asarray(counts) / float(len(timeSeriesData))
-        dist = stats.rv_histogram((counts, edges))
-        self._trainingCDF[target] = dist
+        self._trainingCDF[target] = mathUtils.trainEmpiricalFunction(timeSeriesData, minBins=20)
```

Comment:
The minBins is hard-coded, could you create a variable for it?

Reply:
We discussed opening this as a user-facing variable and decided that there was no benefit at this point; most users are overwhelmed with the ARMA's existing options. See #798. This feature could be added at an appropriate juncture.

Reply:
done